src/js/boot.js.tpl

```diff
@@ -15,14 +15,16 @@ define([], function() {
             // Loading message while we fetch JS / CSS
             el.innerHTML = '<div style="font-size: 24px; text-align: center; padding: 72px 0; font-family: \'Guardian Egyptian Web\',Georgia,serif;">Loading…</div>';
 
-            config = {
-                'assetPath': '<%= assetPath %>',
+            const extraConfig = {
+                'assetPath': '',
```

Comment:
Is the idea that this value will still be populated? I think that variable makes it based on the deployed path in the interactive bucket.